### DISCUSSION TITLE
Change currency from euro to dollars for GitHub sponsors

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#111367] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a bank account with euro and dollars, the payment is in dollars, so it probably be better to get money in dollars (no exchange translation fees).

How can I change the bank account number and currency? (I have separated numbers for PLN, Euro, and Dollars).

**Comment by gil-son** [comment, marked as answer]
Hello, @jcubic 

So, you need to go to GitHub Sponsor - Dashboard - Payouts - Credentials required - Payouts informations - Edit your bank information:

![sponsors](https://github.com/community/community/assets/72712095/07e96f31-fd17-4266-b491-680623d09794)



If I supported you consider to mark my answer. God bless you!

**Reply by jcubic** [reply]
Yes, thanks found it after adding a question. I wanted to delete the question.

**Comment by kissu** [comment]
I am not sure why the accepted answer is marked as spam, but unfortunately, this does not solve the issue of having the invoice in € or any other currency. This issue is [still relevant](https://github.com/orgs/community/discussions/47937) I guess.

I would like to have the equivalent of the 57$ in € here.

<img src="https://github.com/community/community/assets/5133074/68fcfa3c-6168-4966-b4e1-e0c21036212c" width="600" height="auto">

---

PS: also please do not delete the question, what's the point?
